docs/documentation.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Documentation
+---
+
+Once you’re ready to release your webhook feature, you’ll need to add documentation.
+At Svix we consider our documentation as part of the product.
+We put a lot of effort into making sure our documentation makes it as easy as possible for our users to send webhooks reliably at scale.
+
+We've reviewed several webhook services (e.g. Zoom, GitHub, Cloudinary) in a [blog series](https://www.svix.com/blog/tag/documentation/) if you're interested in seeing some examples.
+We also provide a documentation template at [https://dashboard.svix.com/documentation/generator](https://dashboard.svix.com/documentation/generator).
+
+Aside from the basics of how to create a webhook, the best webhook docs we see do 5 things well:
+
+1. Explain why and how to verify webhook signatures (with code samples!).
+2. Provide a comprehensive Event Catalog with detailed explanations and sample payloads of each event type.
+3. Fully explain their retry policy including which status codes will trigger retries, and the actual retry schedule.
+4. Give general troubleshooting tips for initial setup and recovering from endpoint failures.
+5. Give guidance on how to test endpoints.
+
+These 5 things will help your users get setup without a hassle as well as quickly recover from endpoint failures.
+
+We've also created a lot of tools and content to help you write great webhook docs.
+Many users simply link to these materials while others use them as a starting point and customize to their specific use cases:
+
+- Our [webhook verification docs](https://docs.svix.com/receiving/verifying-payloads/why).
+- [Embeddable Event Catalog](https://docs.svix.com/event-types#publishing-your-event-catalog).
+- [Documentation Template & Generator](https://dashboard.svix.com/documentation/generator).
+- [Svix Play](https://www.svix.com/play/): Webhook Tester/Endpoint Generator.
+- Blog post on [writing great webhook docs](https://www.svix.com/blog/documenting-webhooks/).
```

Comment:
Don't use titlecase for these items (you made it title case in some, and normal case in others. Should be normal.